src/ResourceManager/Compute/Commands.Compute/Generated/InvokeAzureComputeMethodCmdlet.cs

```diff
@@ -30,7 +30,7 @@
 
 namespace Microsoft.Azure.Commands.Compute.Automation
 {
-    [Cmdlet(VerbsLifecycle.Invoke, "AzureComputeMethod", DefaultParameterSetName = "InvokeByDynamicParameters")]
+    //[Cmdlet(VerbsLifecycle.Invoke, "AzureComputeMethod", DefaultParameterSetName = "InvokeByDynamicParameters")]
```

Comment:
This is a generated file.  We decided not to generate Invoke-AzureComputeMethod cmdlet, which is a general cmdlet for invoking any compute APIs.  We modified our auto-generator such that it has an option to choose if it generates the cmdlet.  If we don't want, the generator just comments out.